app/Model/UserLoginProfile.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+App::uses('AppModel', 'Model');
+
+class UserLoginProfile extends AppModel
+{
+    public $actsAs = array(
+        'AuditLog',
+            'Containable',
+            'SysLogLogable.SysLogLogable' => array(
+                'userModel' => 'User',
+                'userKey' => 'user_id',
+                'change' => 'full'
+            ),
+            'Containable'
+    );
+
+    public $validate = [
```

Comment:
the data is not made by users, but by the software itself. I will add one validation rule for the `status`. In case of a developer does something wrong.